index.js

```diff
@@ -37,18 +37,23 @@ server.get("/artigo/apagar/:slug", (request, response) => {
 });
 
 server.get("/artigo/:slug", (request, response) => {
-  const slug = request.params.slug;
-
   const database = JSON.parse(jetpack.read('./database.json'));
-  const artigo = database.article[slug];
+  const slug = request.params.slug;
+  const artigo = database.article[slug]
 
-  if (request.headers.accept === "application/json") {
-    response.json({ ...artigo, slug });
+  if(artigo === undefined) {
```

Comment:
Usando o `!` antes da variável é igual comparar `artigo === undefined` ou  `artigo === null` ou  `artigo === false` ou  `artigo === false`, tudo aquilo que seja considerado "FALSO"

```suggestion
  if(!artigo) {
```

Reply:
corrigido.